fuzzer/modules/sql_injection/strategies/blind.py

```diff
@@ -1,9 +0,0 @@
-
-def test_blind_sql_injection():
```

Comment:
Removing this strategy for now. Will look over OWASP documentation and think of other strategies we can do.